Error while installing packages for development environment

running npm install got me into these errors: 
```
4 warnings and 4 errors generated.
make: *** [Release/obj.target/secp256k1/native/addon.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/bassem/.nvm/versions/node/v12.13.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:210:5)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 19.0.0
gyp ERR! command "/Users/bassem/.nvm/versions/node/v12.13.0/bin/node" "/Users/bassem/.nvm/versions/node/v12.13.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/bassem/Desktop/Workspace/Nem.io/forks/nem2-hd-wallets/node_modules/tiny-secp256k1
gyp ERR! node -v v12.13.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
npm WARN nem2-hd-wallets@0.6.0 license should be a valid SPDX license expression

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tiny-secp256k1@1.0.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the tiny-secp256k1@1.0.1 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/bassem/.npm/_logs/2019-11-27T00_47_38_390Z-debug.log
```

a proposed solution for that issue would be adding `"tiny-secp256k1": "^1.1.3" ` to the dependencies.

find logs here -> [attachment](https://drive.google.com/file/d/1l7LLwZVpFxZSVVHa_N2EUtnVgW7XZ7L_/view?usp=sharing)

--- a/package.json
+++ b/package.json
@@ -18,7 +18,8 @@
     "create-hmac": "^1.1.7",
     "js-sha3": "^0.8.0",
     "nem2-sdk": "^0.15.0",
-    "rxjs": "^6.5.3"
+    "rxjs": "^6.5.3",
+    "tiny-secp256k1": "^1.1.3" 
   },
   "devDependencies": {
     "@types/chai": "^4.2.5",

